iot-hub/Samples/device/DeviceReconnectionSample/Program.cs

```diff
@@ -47,10 +51,22 @@ public static async Task<int> Main(string[] args)
                 s_deviceConnectionString = args[0];
             }
 
-            var sample = new DeviceReconnectionSample(s_deviceConnectionString, s_transportType, logger);
+            // Check environment variable for transport type,
+            if (!Enum.TryParse(s_transportType, out TransportType transportType)
```

Comment:
Elevates transport to an environment variable/command-line var.

We really should update these samples to use a command-line library we like, however.

Reply:
Is this case sensitive?

Reply:
Yes

Reply:
It isn't now.